src/blocks/collapsible/collapsible.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * BLOCK: collapsible
+ *
+ * A collapsible content block.
+ */
+
+// External dependencies.
+import classnames from 'classnames';
+
+// Import CSS.
+import './style.scss';
+import './editor.scss';
+
+// Internal dependencies.
+import allowedBlocks from '../../components/allowed-blocks';
+
+// WordPress dependencies.
+const { __ } = wp.i18n;
+const { registerBlockType } = wp.blocks;
+const {
+	PanelBody,
+	ToggleControl,
+	SelectControl,
+	TextControl,
+	RangeControl,
+	Path,
+	SVG,
+	Button,
+	ButtonGroup,
+} = wp.components;
+const {
+	InnerBlocks,
+	BlockControls,
+	InspectorControls,
+	RichText
+} = wp.blockEditor;
+const { select } = wp.data;
+const { getBlocks } = select( 'core/block-editor' );
+
+import HeadingToolbar from '../headline/heading-toolbar';
+
+// Register the block.
+registerBlockType( 'bu/collapsible', {
+	name: 'bu/collapsible',
+	title: __( 'Collapsible' ),
+	description: __( 'A collapsible content block.' ),
+	icon: <SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><Path fill="#c00" d="M19 7h-1V5h-4v2h-4V5H6v2H5c-1.1 0-2 .9-2 2v10h18V9c0-1.1-.9-2-2-2zm0 10H5V9h14v8z"></Path></SVG>,
+	category: 'bu',
+	supports: {},
```

Comment:
@turtlepod Do you think it would make sense to add support for wide & full alignment options if a theme supports them? 

I think left and right would be a bad idea for this type of component, but I could see where users in COM or our other sites might want to make the block wider or full width like they can with other blocks.

Reply:
I think you are right. I updated and add wide&full width support.